packages/salesforcedx-sobjects-faux-generator/src/generator/declarationGenerator.ts

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { ChildRelationship, Field, SObject } from '../types';
+
+export const MODIFIER = 'global';
+export interface FieldDeclaration {
+  modifier: string;
+  type: string;
+  name: string;
+  comment?: string;
+}
+
+export interface SObjectDefinition {
```

Comment:
We could move this and `FieldDeclaration` to the `types` folder

Reply:
Will do,